public/app/features/iam/api/scripts/generate-rtk-apis.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
This could go straight into `scripts/generate-rtk-apis` I think.

Reply:
The schema file can be custom for each client.

Reply:
For example, 
```
    '../public/app/features/provisioning/api/endpoints.gen.ts': {
      apiFile: '../public/app/features/provisioning/api/baseAPI.ts',
      schemaFile: '../data/openapi/provisioning.grafana.app-v0alpha1.json',
      apiImport: 'baseAPI',
      argSuffix: 'Arg',
      responseSuffix: 'Response',
      tag: true,
    },
```

Reply:
ohhhhh nice 🙌 ok, didn't realise that. presumably we can also move query library into the main one as well then?

i'll do that in a follow up PR 👍